presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchConnectorConfig.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.elasticsearch;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableSet;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+import io.airlift.units.MinDuration;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+import java.io.File;
+import java.util.Set;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class ElasticsearchConnectorConfig
+{
+    private String defaultSchema = "default";
+    private Set<String> tableNames = ImmutableSet.of();
+    private File tableDescriptionDir = new File("etc/elasticsearch/");
+    private int scrollSize = 1000;
+    private Duration scrollTime = new Duration(20, SECONDS);
+    private int maxHits = 1_000_000;
+    private Duration requestTimeout = new Duration(10, SECONDS);
```

Comment:
Isn't `10s` for a single request high? If there is a problem, then various thread will block for a long time. What do you think?

Reply:
And why do we have an even larger scroll timeout? Can we set these timeouts to `1s` or sth small?

Reply:
If those values work well in your prod environment I am fine with them.